DuckDuckGo.xcodeproj/project.pbxproj

```diff
@@ -2837,6 +2837,8 @@
 		EE0429E02BA31D2F009EB20F /* FindInPageTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = EE0429DF2BA31D2F009EB20F /* FindInPageTests.swift */; };
 		EE098E772C8EDE2C009EBA7F /* AutofillCredentialsImportManager.swift in Sources */ = {isa = PBXBuildFile; fileRef = EE098E762C8EDE2C009EBA7F /* AutofillCredentialsImportManager.swift */; };
 		EE098E782C8EDE2C009EBA7F /* AutofillCredentialsImportManager.swift in Sources */ = {isa = PBXBuildFile; fileRef = EE098E762C8EDE2C009EBA7F /* AutofillCredentialsImportManager.swift */; };
+		EE20990D2CBE9F4700F90FAE /* AutofillCredentialsImportManagerTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = EE20990C2CBE9F4700F90FAE /* AutofillCredentialsImportManagerTests.swift */; };
```

Comment:
Just FYI, I don’t think the AutofillCredentialsImportManagerTests.swift file was added (which is what is causing the unit tests to fail)

Reply:
Urgh... rebase fail